storage/storage_local.go

```diff
@@ -16,8 +18,15 @@ type storageLocal struct {
 
 // NewStorageLocal : Creates a new Storage module which uses the local disk.
 func NewStorageLocal() Storage {
+	path := `F:\Code\specialedge\storage\java\`
+
+	// If the path doesn't exist, create it.
+	if _, err := os.Stat(path); os.IsNotExist(err) {
+		os.Mkdir(path, 0755)
```

Comment:
I think worth having an info statement here if the directory's being created.

Reply:
Agreed.